Scripts/script-ParseEmailHeaders.yml

```diff
@@ -0,0 +1,38 @@
+commonfields:
+  id: ParseEmailHeaders
+  version: -1
+name: ParseEmailHeaders
+script: |-
+  from email.parser import HeaderParser
+
+  ec = {"Email.Headers" : {}}
```

Comment:
`"` -> `'`

Reply:
fixed

Reply:
Fixed back.
In Python it should be " and not '